### ISSUE TITLE
Does this reporter support the newest node version?

### ISSUE THREAD

**Issue by raypettersen** [voxpupuli/puppet-prometheus_reporter#9]
I'm getting a lot of errors scraping the .prom files using the new version of the node_exporter.

Example:

time="2017-03-23T12:57:35+01:00" level=error msg="error gathering metrics: 20 error(s) occurred:
* gathered metric family puppet_report_resources has help "Metric read from /etc/puppet/metrics/foo.prom" but should have "Metric read from /etc/puppet/metrics/bar.prom"

Note that the error involves two different prom files. It seems the first prom (alfabetically) actually succeeds, while everything else fails.

**Comment by raypettersen** [comment]
Is this project maintained? None of the other posts have any kind of interaction?

**Comment by marius-cotofana-hs** [comment]
Seems to be related to https://github.com/prometheus/node_exporter/issues/485

A quick fix would be to add:

```
       file.write("# HELP puppet_report metric_help\n")
       metrics.each do |metric, data|
         file.write("# HELP puppet_report_#{metric} metric_help\n")
       end
```

to `prometheus.rb` line 71

**Comment by raypettersen** [comment]
So if I understand this correctly, the bug is in node, but a workaround in this reporter is possible with your suggestion?

**Comment by marius-cotofana-hs** [comment]
That's right.

**Comment by pugnacity** [comment]
@marius-cotofana-hs could you make a pull request for the workaround?

**Comment by oleg-glushak** [comment]
@pugnacity https://github.com/voxpupuli/puppet-prometheus_reporter/pull/22

**Comment by LDaneliukas** [comment]
Should this be closed as #22 was merged?

**Comment by LDaneliukas** [comment]
Verified running the reporter with `node-exporter` `0.16.0-8` - everything works as expected, this can be closed
